services/tenant-ui/jest.config.ts

```diff
@@ -162,9 +162,7 @@ export default {
   // ],
 
   // An array of regexp pattern strings that are matched against all test paths, matched tests are skipped
-  // testPathIgnorePatterns: [
-  //   "/node_modules/"
-  // ],
+  testPathIgnorePatterns: ["/frontend/"],
```

Comment:
Are we ACTUALLY supposed to skip tests in frontend? 🤔

Reply:
There are two testing frameworks in place. The "backend" was already using Jest while the frontend is using Vitest based on the suggestion by @loneil. The GHA run tests for both separately. If I don't ignore the frontend module when running the Jest tests, those tests will fail because Vitest is not installed or configured. We could align and use the same testing library for both.

This also brings up a larger question I have about why the tenant ui backend has the frontend nested within it? Why not have the backend and frontend folders setup as siblings?

Reply:
Oh yeah that's just the pattern I've always used to do, probably no specific good reason to so we could reconfigure I'm sure at some point easily enough